gateway/configuring-license/README.md

```diff
@@ -0,0 +1,130 @@
+# Configuring Licenses for Confluent Gateway on Kubernetes
```

Comment:
Optional: For certificate generation, we have the example in `/gateway/certificates`. We may want to do something similar for licenses also: `/gateway/licenses` or `/gateway/license-configuration`